extensions/terminal-suggest/src/test/completions/upstream/ls.test.ts

```diff
@@ -66,12 +66,10 @@ export const lsTestSuiteSpec: ISuiteSpec = {
 
 		// Basic options
 		// TODO: The spec wants file paths and folders (which seems like it should only be folders),
-		//       but neither are requested https://github.com/microsoft/vscode/issues/239606
 		{ input: 'ls |', expectedCompletions: allOptions, expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
 		{ input: 'ls -|', expectedCompletions: allOptions, expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
 
-		// Filtering options should request all options so client side can filter
-		{ input: 'ls -a|', expectedCompletions: allOptions, expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
```

Comment:
If this is what fig gives us, we might need to make `-` a trigger character so that the list is refreshed after `-` and when one is deleted. https://github.com/microsoft/vscode/issues/240034